release-notes/EPM/I-DOCSIS/EPM_7.0.1_I-DOCSIS.md

```diff
@@ -0,0 +1,36 @@
+---
+uid: EPM_7.0.1_I-DOCSIS
+---
+
+# EPM 7.0.1 I-DOCSIS - preview
+
+> [!IMPORTANT]
+> We are still working on this release. Some release notes may still be modified or moved to a later release. Check back soon for updates!
+
+## New features
+
+#### Channel Utilization added for US/DS Service Group [ID_38849]
+
+The ​Upstream and Downstream Service Group entities now display the Channel Utilization average of the associated channels in the data section.
```

Comment:
@SkylineGBO, right now this is still a bit vague, so users might not be sure where to find this change. Where can they see this in the UI (on which page, with which parameters)?